### PR TITLE
speedup build (sbt-coursier, sbt-updates) & minor lib updates 

### PR DESCRIPTION
**Pull request by antonkulaga**
I've made minor dependency updates and also updated sbt to 0.13.13 as 0.13.8 is too slow.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -17,10 +17,26 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-coffeescript" % "1.0.0")
...
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-stamp" % "5.2.0")
+
+/**
+  * coursier
+  *
+  * better and faster dependency resolution
+  * @see https://github.com/alexarchambault/coursier
+  */
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15-1")
```
Comment (vidma) [inline project/plugins.sbt]: nice! 👏 

> no global lock (no "Waiting for ~/.ivy2/.sbt.ivy.lock to be available").
> better offline mode - one can safely work with snapshot dependencies if these are in cache (SBT tends to try and fail if it cannot check for updates),
> downloading of artifacts in parallel,

did you use it already for a while? no issues noticed?

Comment (vidma) [inline project/plugins.sbt]: hmm not sure if Travis builds got a big speedup. maybe we need to re-enable the caching of certain dirs if there's no messy global lock anymore.

i.e. `The default global cache used by coursier is ~/.coursier/cache/v1`

Comment (antonkulaga) [inline project/plugins.sbt]: @vidma I've been using coursier for several months in all of my projects and never had issues with it.

Comment (antonkulaga) [inline project/plugins.sbt]: @vidma I've tried to add caching configuration to travis, but it did not change anything in terms of build times

Comment (antonkulaga) [inline project/plugins.sbt]: Anyway it is still faster than ivy, it took 6.55 to build for 2.11.8 while in your previous build it took >11 minutes for each scala version.

Comment (vidma) [inline project/plugins.sbt]: cool. it might also be that caching is done for master only... so we'll later see if it helps...

**Comment by vidma** [comment]
👍

**Comment by vidma** [comment]
let's try to 🐑 this.

**Comment by vidma** [comment]
cell execution seems failing after this PR (2.1.0 and maybe 2.0.0 as well):
```
<console>:12: error: not found: value globalScope
       globalScope.sparkContext.setJobGroup("cell-CF7654DF29474E5384025A6F49FCCC95", "run-1485946611387: val sqlContext = new org.apache.spark.sql.SQLContext(sc)import sqlContext.implicits._")
       ^
```

needs more investigation.

**Comment by vidma** [comment]
reverted for now https://github.com/andypetrella/spark-notebook/commit/5f3fca400ecff612504c9d9d3341188a3c7f38e0

will investigate this later.

**Comment by alexarchambault** [comment]
`globalScope` missing seems to mean that your [initialization script](https://github.com/andypetrella/spark-notebook/blob/master/conf/scripts/init.sc) didn't run properly. There must have been errors prior to this one somewhere...

**Pull request merged**